gitlab/v4/objects/iterations.py

```diff
@@ -16,11 +16,31 @@ class GroupIterationManager(ListMixin, RESTManager):
     _path = "/groups/{group_id}/iterations"
     _obj_cls = GroupIteration
     _from_parent_attrs = {"group_id": "id"}
-    _list_filters = ("state", "search", "include_ancestors")
+    # The "in" keyword collides with python's "in" keyword raising a SyntaxError.
+    # For this reason, we have to use the query_parameters argument:
+    # group.iterations.list(query_parameters={"in": "title"})
+    _list_filters = (
+        "include_ancestors",
+        "include_descendants",
+        "search",
+        "state",
+        "updated_after",
+        "updated_before",
+    )
 
 
 class ProjectIterationManager(ListMixin, RESTManager):
     _path = "/projects/{project_id}/iterations"
     _obj_cls = GroupIteration
     _from_parent_attrs = {"project_id": "id"}
-    _list_filters = ("state", "search", "include_ancestors")
+    # The "in" keyword collides with python's "in" keyword raising a SyntaxError.
+    # For this reason, we have to use the query_parameters argument:
+    # group.iterations.list(query_parameters={"in": "title"})
```

Comment:
```suggestion
    # project.iterations.list(query_parameters={"in": "title"})
```

Reply:
from the test that I've conducted on my istance, the right query here would be:

```suggestion
    # project.iterations.list(query_parameters={"in[]": "title"})
```

does it sounds good for you?

Reply:
@nejch thanks for all the feedback, sorry, I totally forgot the CLI ^^

a last point here and then I can make a final squash before the merge

Reply:
I'll lint also in that step

Reply:
> from the test that I've conducted on my istance, the right query here would be:
> 
> does it sounds good for you?

@cristianocasella I think we shouldn't manually add brackets- instead we should make `in` an `ArrayAttribute` and python-gitlab will do that on its own already. But we can do this in a follow-up MR as well.